src/ceph-volume/ceph_volume/util/device.py

```diff
@@ -1,13 +1,14 @@
 # -*- coding: utf-8 -*-
-
+# type: ignore
```

Comment:
why do we need this here?

Reply:
To be honest: it's more a legacy than anything else, I was just annoyed because my IDE was full of red (full of mypy warnings) while I was writing this patch.
For what it is worth, I have a branch for addressing type annotations, but it's tough work....